src/patchkernel/patch_kernel_parallel.cpp

```diff
@@ -2947,6 +3234,54 @@ std::vector<int> PatchKernel::getNeighbourRanks()
 	return neighRanks;
 }
 
+/*!
+	Gets a constant reference to the ghost targets needed for data exchange.
+
```

Comment:
A comment here can help to understand better what are ghost targets. Something like:

> For each source rank (key of the returned map) the ordered list of the local ids of the ghost vertices is provided. These vertices are involved as targets in a send/receive data exchange. Targets ghost vertices on a receiving rank have the same order of the source vertices on the related sending rank.

Maybe, it can be written more clearly....